app/views/profiles/_edit.haml

```diff
@@ -9,10 +9,22 @@
   = render 'profiles/edit_public', :profile => profile, :aspect => aspect, :person => person
 
   %hr
+  :javascript
```

Comment:
You will have to define that in app/assets/javascripts/app/router.js. It might be ok to add this to the settings page view (app/assets/javascripts/app/pages/settings.js) but you will definitely have to add the correct route in router.js.
```js
...
 "profile/edit": "settings",
...
```